### PR TITLE
Update streaming-api-storage.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the navigation to copy the resource ID

### CODE REVIEW

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10024

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender/streaming-api-storage.md [hunk 1]
```diff
@@ -56,9 +56,11 @@ Once the Storage account is created you will need to:
 4. In the **Add new Streaming API settings** flyout that appears, configure the following settings:
    1. **Name**: Choose a name for your new settings.
    2. Select **Forward events to Azure Storage**.
-   3. In the **Storage Account Resource ID** box that appears, type your **Storage Account Resource ID**. To get your **Storage Account Resource ID**, open the Azure portal at <https://portal.azure.com>, click **Storage accounts** \> go to the properties tab \> copy the text under **Storage Account Resource ID**.
+   3. To display the Azure Resource Manager resource ID for a storage account in the Azure portal, follow these steps:
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender/streaming-api-storage.md]:
```suggestion
4. To display the Azure Resource Manager resource ID for a storage account in the Azure portal, follow these steps:
```

File: microsoft-365/security/defender/streaming-api-storage.md [hunk 2]
```diff
@@ -56,9 +56,11 @@ Once the Storage account is created you will need to:
...
 
-      :::image type="content" source="../defender-endpoint/images/storage-account-resource-id.png" alt-text="A Storage Account Resource ID" lightbox="../defender-endpoint/images/storage-account-resource-id.png":::
+1. Navigate to your storage account in the Azure portal.
+1. On the **Overview** page, in the **Essentials** section, select the **JSON View** link.
+1. The resource ID for the storage account is displayed at the top of the page, copy the text under **Storage Account Resource ID**.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender/streaming-api-storage.md]:
```suggestion
   1. Navigate to your storage account in the Azure portal.
   2. On the **Overview** page, in the **Essentials** section, select the **JSON View** link.
   3. The resource ID for the storage account is displayed at the top of the page, copy the text under **Storage Account Resource ID**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @msbemba, kindly look at the preview to make sure that the formatting is correct. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Pull request merged**